CE558_Team2/points_node[]  nodes
